transcode_system_migration.sql

```diff
@@ -351,41 +382,52 @@ BEGIN
             WHEN @audioBitrate < 192000 THEN 4
             ELSE 2
         END;
-        SET @details = '{"type":"Mp3AudioFormat",' + 
-                        '"CompressionLevel":' + CONVERT(NVARCHAR(10), @compressionLevel) + '}';
+        SET @details = '{"type":"Mp3AudioFormat",' +
+                        '"CompressionLevel":' + CONVERT(NVARCHAR(10), @compressionLevel) + ',' +
+                        '"Bitrate":' + CONVERT(NVARCHAR(10), @audioBitrate) +
+                        '}';
     END
     ELSE IF @extension='avi'
     BEGIN
         SET @details = '{"type":"AviVideoFormat",' +
                         '"BackgroundColor":"#00000000",' +
-                        '"CompressionLevel":23,' + 
+                        '"CompressionLevel":23,' +
                         '"Height":' + CONVERT(NVARCHAR(10), @height) + ',' +
                         '"Width":' + CONVERT(NVARCHAR(10), @width) + ',' +
                         '"ResizeMode":0,' + -- fixed size
                         '"BackgroundWidth":0,' +
-                        '"BackgroundHeight":0}'
+                        '"BackgroundHeight":0,' +
+                        '"AudioBitrate":' + CONVERT(NVARCHAR(10), @audioBitrate) + ',' +
+                        '"VideoBitrate":' + CONVERT(NVARCHAR(10), @videoBitrate) +
```

Comment:
I'm pretty sure that these need to be after the `CompressionLevel` parameter with `VideoBitrate` before `AudioBitrate`.
Otherwise, the fingerprints won't match and the migrated renditions will be re-generated.
Yes, this is not very resilient, but that's unfortunately the way it currently is :/

We should test that renditions are not re-generated after having been migrated.

Reply:
@zlepper I think you might have missed this comment? 🤔

Reply:
You are right, something is not adding up here. Apparently I was a bit too fast on the rendition i looked at as part of this....